controllers/admin/AdminLinkWidgetController.php

```diff
@@ -218,11 +219,45 @@ protected function buildHelper()
         return $helper;
     }
 
-    public function validateForm($data)
+    /**
+     * @param array $fields
+     * @return bool
+     */
+    public function validateLinkName(array $fields)
```

Comment:
Ok, but all what you need to do is to add required in ObjectModel and required in Form to have information that field is required without throwing an Exception, you just forgot to add required in Form, that's it